apps/frontend-manage/src/components/evaluation/EvaluationUnavailableNotification.tsx

```diff
@@ -12,6 +29,68 @@ function EvaluationUnavailableNotification() {
         }}
         message={t('manage.evaluation.evaluationNotYetAvailable')}
       />
+
+      <div className="mt-4 flex flex-row gap-8 text-xs text-gray-500">
+        <div className="space-y-2">
+          {elementName && (
+            <div>
+              <span className="font-semibold">
+                {t('manage.evaluation.elementName')}
+              </span>
+              : {elementName}
+            </div>
+          )}
+          {elementType && (
+            <div>
+              <span className="font-semibold">
+                {t('manage.evaluation.elementType')}
+              </span>
+              : {elementType}
```

Comment:
We should definitely use the properly translated strings here instead of the GraphQL enums

```suggestion
              : {t(`shared.${elementType}.typeLabel`)}
```